front/app/components/IdeaCard/index.tsx

```diff
@@ -125,7 +125,7 @@ const IdeaCard = memo<IdeaCardProps>(
     const handleClick = (e: React.MouseEvent) => {
       e.preventDefault();
       updateSearchParams({ scroll_to_card: idea.data.id });
-      clHistory.push(`/ideas/${slug}${params}`);
+      clHistory.push(`/ideas/${slug}${params}?go_back=true`);
```

Comment:
Could you explain how this one works? Can't seem to find where/how this is implemented in our codebase.

Reply:
Essentially it's just used to help determine whether the goBack button should use clHistory.goBack, or should push a more general URL (E.g. The project page or homepage).

In the DesktopTopBar you can see handling related to this:
```
  const handleGoBack = useCallback(() => {
    if (goBack) {
      clHistory.back();
    } else if (project) {
      clHistory.push(`/projects/${project.attributes.slug}`);
    } else {
      clHistory.push('/');
    }
  }, [goBack, project]);
```